Test Policy

```diff
@@ -0,0 +1,66 @@
+Purpose and Scope 
+The purpose of this document is to communicate at a high level the policy for UKHO for testing of new software, updates, hardware and other related changes.
+
+Why do we Test?
+•	To build confidance in our systems
+•	Detect defects
+•	Generate information
+•	Manage quality risks
+
+How do we measure the effectives of testing?
+Delivery to customers of software and hardware that provides business value
+
+UKHO Long Term Testing Goals
+•	To move to a Devops Model
+•	Test Automation provides the bulk of testing functionality
+•	High Value manual testing is used 
+•	Testing across UKHO is done within an agile framework
+
+Who Tests?
+Test Engineering line managers a group of Test Engineers, ranging from junior to lead people, they provide testing capability across most of UKHO where testing is carried out within scrum teams.
+
+Outside of the scrum teams other people do test, for instance within Operations testing of hardware is carried out by the relevant SME, like network engineers carry out network testing. In some non-agile teams Business Analysts also carry out testing.
+
+Who Manages Testing?
+
+The Head of Test line manages all test engineers, but test engineers are tasked managed by the overall technology program in conjunction with the Head of Test.
+
+Other areas of the business carry out various testing activities, but these people will not necessarily be professional software testers, they are line managed by their relevant program.
+
+Roles and Responsibilities of Test Engineering
+
+This will only deal with the roles within Test Engineering
+
+http://technology/TPE/TE/TEEnterpriseWiki/Pages/Test%20Engineering%20Roles%20and%20Responsbilities.aspx
+In Scope 
+
+•	All software applications
+•	Any changes to database structures
+•	Network Changes
+•	Hardware Changes
+•	3rd Party Software
+•	Cloud Based Applications
+
+
+The Testing Lifecycle
+
+UKHO follows the Disciplined Agile Delivery(DAD) and testing forms part of that lifecycle process which is collectively referred to as the Agile Delivery Governance Framework.
+
+Testing will be done at all stages of the development of software and products
+
+There are sub policies that cover the Agile and Non-Agile methods as well as Security, Penetration , Load and Performance testing,they can be found here.
+
+http://technology/TPE/TE/TEEnterpriseWiki/Pages/Test%20Engineering%20Guidance.aspx
```

Comment:
You won't be able to link back to SharePoint, and we probably shouldn't be exposing internal paths on GitHub !?

Reply:
Good point, I will remove that link